Lib/test/test_functools.py

```diff
@@ -27,8 +27,7 @@
 
 py_functools = import_helper.import_fresh_module('functools',
                                                  blocked=['_functools'])
-c_functools = import_helper.import_fresh_module('functools',
-                                                fresh=['_functools'])
+c_functools = import_helper.import_fresh_module('functools')
```

Comment:
I create an seperated PR in https://github.com/python/cpython/pull/23407.
I update this test_functools.py just for using CI to test this  PR.